pkg/kubectl/cmd/util/cached_discovery.go

```diff
@@ -69,7 +69,6 @@ func (d *CachedDiscoveryClient) ServerResourcesForGroupVersion(groupVersion stri
 	if err == nil {
 		cachedResources := &metav1.APIResourceList{}
 		if err := runtime.DecodeInto(api.Codecs.UniversalDecoder(), cachedBytes, cachedResources); err == nil {
-			glog.V(6).Infof("returning cached discovery info from %v", filename)
```

Comment:
why delete those?

Reply:
It overwhelms debug output for most operations

Reply:
maybe increase the level? otherwise we readd them next time we need that info.

Reply:
updated verbosity level instead